### PR TITLE
LS-2217 Create Gradle Dependency Graph

### PR DESCRIPTION
**Pull request by andyloughran**
## Proposed changes

Testing a shared workflow.

### What changed

Used the .github/workflow-templates to configure a template for submitting gradle dependencies to GHAS.

### CODE REVIEW

File: .github/workflows/gradle_graph.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Dependency Submission
+
+on:
+  push:
```
Comment (andyloughran) [inline .github/workflows/gradle_graph.yml:4]:
```suggestion
  workflow_dispatch:
  push:
```

File: .github/workflows/gradle_graph.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Dependency Submission
+
+on:
+  workflow_dispatch:
+  push:
```
Comment (andyloughran) [inline .github/workflows/gradle_graph.yml:4]:
```suggestion
  pull-request:
```

File: .github/workflows/gradle_graph.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Dependency Submission
+
+on:
+  workflow_dispatch:
+  pull-request:
```
Comment (andyloughran) [inline .github/workflows/gradle_graph.yml]:
```suggestion
  pull_request:
```

File: .github/workflows/gradle_graph.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+name: Dependency Submission
+
+on:
+  workflow_dispatch:
+  pull_request:
```
Comment (andyloughran) [inline .github/workflows/gradle_graph.yml]:
```suggestion
  push:
```

File: .github/workflows/gradle_graph.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+name: Dependency Submission
+
+on:
+  workflow_dispatch:
```
Comment (andyloughran) [inline .github/workflows/gradle_graph.yml]:
```suggestion
```

**Review by paroxp — APPROVED** [review]
Shame about commit history. But here we go.

**Pull request merged**